iam/api/QuickStartNewTest/QuickStartNewTest.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Iam.v1;
+using Google.Apis.Iam.v1.Data;
+using Xunit;
+
+public class QuickStartNewTest : IDisposable

```

Comment:
Same comment applies here as to how best name these classes, files and projects.